src/main/java/bio/terra/common/stairway/StairwayJdbcConfiguration.java

```diff
@@ -0,0 +1,11 @@
+package bio.terra.common.stairway;
+
+import bio.terra.common.db.JdbcConfiguration;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
I want to consolidate this into the parent class and rename that, as I'm not interested in it being the same code as in buffer service. That is, I don't want to be in the business of prescribing JDBC connecting for other services yet.